easybuild/tools/filetools.py

```diff
@@ -171,6 +171,34 @@ def write_file(path, txt, append=False, forced=False):
         raise EasyBuildError("Failed to write to %s: %s", path, err)
 
 
+def readlink(symlink_path):
+    """
+    Read symlink target at the specified path to the given path.
+
+    :param symlink_path: symlink file path
+    """
+
+    # note: we can't use try-except-finally, because Python 2.4 doesn't support it as a single block
```

Comment:
minimal required version is Python 2.6 now, so you can use `try`/`except`/`finally`

but, not relevant here, so please just drop this comment